.github/workflows/release.yml

```diff
@@ -38,9 +38,9 @@ jobs:
         uses: github-actions-x/commit@v2.9
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          push-branch: 'master'
-          commit-message: 'update changelog'
-          force-add: 'true'
+          push-branch: master
+          commit-message: update changelog
+          force-add: "true"
```

Comment:
are quotes here neccessary?

Reply:
I changed all occurences and usages to bools.